packages/starknet-snap/src/ui/fragments/FeeTokenSelector.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Card,
+  Field,
+  Form,
+  Selector,
+  SelectorOption,
+  type SnapComponent,
+} from '@metamask/snaps-sdk/jsx';
+
+import { FeeToken, FeeTokenUnit } from '../../types/snapApi';
+
+/**
+ * The props for the {@link FeeTokenSelector} component.
+ *
+ * @property selectedToken - The currently selected fee token.
+ */
+export type FeeTokenSelectorProps = {
+  selectedToken: FeeToken;
```

Comment:
```suggestion
  selectedFeeToken: FeeToken;
```
apply to all name below